CHANGELOG.md

```diff
@@ -3,7 +3,47 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v2.0.0dev - [unreleased]
+## [v2.0.0](https://github.com/nf-core/pairgenomealign/releases/tag/2.0.0) "Naga imo" - [January 27th, 2025]
+
+### Backwards-incompatible changes
+
+- The LAST software was updated and it has new defaults for some of its
+  parameters. The alignments ran with this pipeline will not be identical to
+  the ones from older versions.
+
+### New features
+
+- The `alignment/lastdb` directory is not output anymore. It consumed space,
+  is not usually needed for downstream analysis, and can be re-computed
+  identically if needed.
+- The _many-to-one_ alignment file is not output anymore by default, to save
+  space. To keep this file, you can run the pipeline in `many-to-many` mode
+  with the `--m2m` parameter.
+- The `--seed` parameter allows for all the existing values in the `lastdb`
+  program.
+- Errors caused by absence of alignments at training or plotting steps
+  are now ignored.
+- New parameter `--export_aln_to` that creates additional files containing
+  the alignments in a different format such as Axt, Chain, GFF or SAM.
+
```

Comment:
I couldnt see announcement the additional param change (-p 100000) with CUTN_TARGET here.

Reply:
I do not mention the param change because it is the fix for https://github.com/nf-core/pairgenomealign/issues/18; it is not a user-facing parameter.